### ISSUE TITLE
IntRange annotation defaults wrong for non-Long primitive wrapper types

### ISSUE THREAD

**Issue by astiob** [typetools/checker-framework#2815]
When written as `@IntRange(from=0)` or `@IntRange(to=42)`, the other parameter is defaulted to the minimum/maximum value allowed for the type that is being annotated.

#1300 made it so that the correct extreme values are chosen for non-long primitive types: `int`, `short`, `byte`. However, their corresponding wrapper types still get `Long.MIN_VALUE` and `Long.MAX_VALUE` defaults, which is wrong.

Simple test:
```java
void good(@IntRange(from=0, to=Integer.MAX_VALUE) Integer baz,
          @IntRange(from=0) int quux) {
    quux = baz;
}
void bad(@IntRange(from=0) Integer foo, @IntRange(from=0) int bar) {
    bar = foo;
}
```
```
error: [argument.type.incompatible] incompatible types in argument.
    bar = foo;
          ^
  found   : @IntRange(from=0, to=9223372036854775807) Integer
  required: @IntRange(from=0, to=2147483647) int
```

Tested with Gradle plugin 0.4.5 and Checker Framework 2.11.1 (overridden manually from the plugin’s default 2.11.0).
